src/loader.py

```
@@ -360,13 +450,13 @@ def error_handler(future):
 
     # some logging
     return str(dataset_base_path)
-    
+
 
 def _rio_clip_raster(file_name: str, reference_area: gpd.GeoDataFrame, base_path: Path, out_name: str = None, touched: bool = False):
```

Ruff (0.14.5)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)